Source/GameView/Repositories/GameAPIRepo.swift

```diff
@@ -154,11 +155,21 @@ public class GameAPIRepo: GameRepository {
       return
     }
 
-    guard let priceOverview = gameDetailsData["price_overview"] as? [String: Any] else {
-      return
+    if let priceOverview = gameDetailsData["price_overview"] as? [String: Any] {
```

Comment:
Fine for now but in future keep your keys in a struct

Reply:
Aye, will do. But uh, just a word of warning. Because I did this PR waaaay to early there will be notifications as I fix the CI/CD pipeline for this repo.

Reply:
Haha no worries